Source/Scene/Model.js

```diff
@@ -1516,6 +1565,122 @@ define([
         return getStringFromTypedArray(loadResources.getBuffer(bufferView));
     }
 
+    function replaceAllButFirstInString(string, find, replace) {
+        return string.replace(new RegExp(find, 'g'), function(match, offset, all) {
+            return all.indexOf(find) === offset ? match : replace;
```

Comment:
not sure if it will make a notable difference, but one could cache the position of the first occurence here (to avoid repeated execution of indexOf)